test/WebSites/RoutingWebSite/Controllers/RouteDataController.cs

```diff
@@ -6,16 +6,16 @@
 using Microsoft.AspNetCore.Mvc.Controllers;
 using Microsoft.AspNetCore.Mvc.Filters;
 
-namespace BasicWebSite
+namespace RoutingWebSite.Controllers
```

Comment:
Moved to routing website so the tests are run with both routing and dispatching.